backend/src/services/secret-v2-bridge/secret-v2-bridge-service.ts

```diff
@@ -651,6 +652,56 @@ export const secretV2BridgeServiceFactory = ({
     return count;
   };
 
+  const $getSecretsByFolderMappings = async (
```

Comment:
I see your exporting this function. Then i gues you can remove the dollar symbol. The dollar symbol means its internal only and not exported